Sql_Assignment

```diff
@@ -0,0 +1,310 @@
+| 1064     | 01                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_5 | Demo Product Unit Five  | 3.000000 |    12.07000 |
+| 1064     | 02                | PRODUCT_ORDER_ITEM | 02                | DEMO_UNIT_2 | Demo Product Unit Two   | 2.000000 |     6.33000 |
+| 1102     | 01                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_1 | Demo Product Unit One   | 3.000000 |    16.99000 |
+| 1102     | 02                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_2 | Demo Product Unit Two   | 3.000000 |    18.99000 |
+| 1102     | 03                | RENTAL_ORDER_ITEM  | 01                | NULL        | Rental Item             | 3.000000 |     2.33000 |
+| 1102     | 04                | PRODUCT_ORDER_ITEM | 02                | DEMO_UNIT_3 | Demo Product Unit Three | 1.000000 |    14.99000 |
+| 1153     | 01                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_3 | Demo Product Unit Three | 1.000000 |    14.99000 |
+| 1153     | 02                | RENTAL_ORDER_ITEM  | 01                | NULL        | Rental Item             | 1.000000 |     1.33000 |
+| 1153     | 03                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_4 | Product Unit FourDemo   | 1.000000 |    12.99000 |
+| 1153     | 04                | RENTAL_ORDER_ITEM  | 01                | NULL        | Rental Item             | 1.000000 |     2.33000 |
+| 1255     | 01                | PRODUCT_ORDER_ITEM | 01                | DEMO_UNIT_4 | Demo Product Unit Four  | 3.000000 |    12.99000 |
+| 1255     | 02                | RENTAL_ORDER_ITEM  | 01                | NULL        | Rental Item             | 3.000000 |     1.33000 |
+| 1255     | 03                | PRODUCT_ORDER_ITEM | 02                | DEMO_UNIT_5 | Demo Product Unit Five  | 3.000000 |    10.99000 |
++----------+-------------------+--------------------+-------------------+-------------+-------------------------+----------+-------------+
+
+
+2).List the Person details for all females.
+
+select * from Person where Gender='F';
++-----------+------------+-------------+-----------+--------+------------+---------------------------+-------------------+
+| PARTY_ID  | FIRST_NAME | MIDDLE_NAME | LAST_NAME | GENDER | BIRTH_DATE | EMPLOYMENT_STATUS_ENUM_ID | OCCUPATION        |
++-----------+------------+-------------+-----------+--------+------------+---------------------------+-------------------+
+| CustDemo3 | Lily       | J           | Brown     | F      | 1998-06-11 | EMPS_PARTTIME             | Software Engineer |
+| CustDemo4 | John       | J           | Norvig    | F      | 1998-06-11 | EMPS_FULLTIME             | Software Engineer |
++-----------+------------+-------------+-----------+--------+------------+---------------------------+-------------------+
+
+
+3). List the OrderHeader table contents so that the latest placed order appears in the top.
+
+select * from Order_Header order by Order_Id desc;
```

Comment:
Why order by order_id, it can be different. order_date is the one that tells us which order came when.

Reply:
SELECT * FROM order_header ORDER BY order_date DESC;